examples/next-minimal-starter/src/pages/index.tsx

```diff
@@ -1,18 +1,27 @@
+import React from 'react';
 import { trpc } from '../utils/trpc';
 
 export default function IndexPage() {
-  const hello = trpc.proxy.hello.useQuery({ text: 'client' });
-  if (!hello.data) {
-    return (
-      <div style={styles}>
-        <h1>Loading...</h1>
-      </div>
-    );
-  }
+  const ac = new AbortController();
+  const utils = trpc.proxy.useContext();
+
+  const longQuery = trpc.proxy.longQuery.useQuery('Julius', {
+    trpc: { signal: ac.signal },
+  });
```

Comment:
This shouldn't be necessary for cancelling React Query queries, there's no difference between cancelling and aborting.